rosette/src/BigBang.cc

```diff
@@ -544,6 +544,7 @@ std::tuple<int, bool> BigBang(int argc, char** argv, char** envp) {
         Define("envp", GetEnvp(envp));
         Define("flag-enable-repl", GetReplFlag());
         Define("flag-verbose", VerboseFlag ? RBLTRUE : RBLFALSE);
+        Define("flag-deferLookup", DeferLookupFlag ? RBLTRUE: RBLFALSE);
```

Comment:
I think to maintain Lisp style, it should be `flag-defer-lookup`.

Reply:
Done